bindings/python/crocoddyl/multibody/impulse-costs/com-position.hpp

```diff
@@ -0,0 +1,58 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2019, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef BINDINGS_PYTHON_CROCODDYL_MULTIBODY_IMPULSE_COSTS_COM_POSITION_HPP_
+#define BINDINGS_PYTHON_CROCODDYL_MULTIBODY_IMPULSE_COSTS_COM_POSITION_HPP_
+
+#include "crocoddyl/multibody/impulse-costs/com-position.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+namespace bp = boost::python;
+
+void exposeImpulseCostCoM() {
+  bp::class_<ImpulseCostModelCoM, bp::bases<ImpulseCostModelAbstract> >(
+      "ImpulseCostModelCoM", bp::init<boost::shared_ptr<StateMultibody>, boost::shared_ptr<ActivationModelAbstract> >(
+                                 bp::args("self", "state", "activation"),
+                                 "Initialize the CoM position cost model.\n\n"
```

Comment:
Improve documentation, e.g. "Initialize the CoM position cost model under impulse dynamics.\n\n